src/types.rs

```diff
@@ -95,7 +95,7 @@ impl fmt::Debug for DhtId {
 }
 
 // lhs is closer to target than rhs
-pub fn closer_to_target(lhs: &DhtId, rhs: &DhtId, target: &DhtId) -> bool {
+pub fn closer_to_target(lhs: &NameType, rhs: &NameType, target: &NameType) -> bool {
```

Comment:
function `closer_to_target` implemented for NameType is not used; routing_table calls free function